File: ruby/fizz_buzz/fizz_buzz.rb
# frozen_string_literal: true

# Use:
# 1.upto(100) do |v|
#   p fizz_buzz(v)
# end
def fizz_buzz(value)
  word = ''
  word << 'fizz' if (value % 3).zero?
  word << 'buzz' if (value % 5).zero?
  word << value.to_s if word.empty?
  word
end

File: python/fizz_buzz/fizz_buzz.py
def fizz_buzz(value):
    word = ''
    if value % 3 == 0:
        word += 'fizz'
    if value % 5 == 0:
        word += 'buzz'
    return word or value

#for i in range(0, 100):
#    print fizz_buzz(i)
